### DISCUSSION TITLE
Statamic + Jetstream?

### DISCUSSION THREAD

**Discussion by digitalam025** [statamic/cms#4528] [Q&A] [answered]
Hey there! Simple question.

Is it a good idea to install Statamic into a Jetstream app? Do any problems come to mind from your fingertips?

Thanks!

**Comment by xmrtni** [comment]
You might want to give it a try as my situation may be unique, but I've had nothing but issues - from routing to authentication.

**Reply by edalzell** [reply]
> You might want to give it a try as my situation may be unique, but I've had nothing but issues - from routing to authentication.

Have you opened any issues for these?

**Reply by xmrtni** [reply]
No, not yet. I wanted to set up a new project to test first in order to rule out the possibility of it being an issue within my app.

**Reply by digitalam025** [reply]
I see, guess I'll give it a shot, thanks!

**Comment by edalzell** [comment, marked as answer]
Statamic does support having users in the Laravel standard `users` table: https://statamic.dev/knowledge-base/storing-users-in-a-database#in-an-existing-laravel-app so it should work.

**Reply by digitalam025** [reply]
Alright thanks!
